pulumi/sitecustomize.py

```
@@ -0,0 +1,9 @@
+"""Enable coverage collection in subprocesses when configured."""
+
+try:
+    import coverage
+except Exception:  # pragma: no cover - best-effort only
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)